packages/fhir-team-management/src/components/OrganizationAffiliation/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Row, Col, Spin, PageHeader } from 'antd';
+import { FHIRServiceClass, BrokenPage, Resource404 } from '@opensrp/react-utils';
+import { useQuery } from 'react-query';
+import AffiliationTable from './Table';
+import { IBundle } from '@smile-cdr/fhirts/dist/FHIR-R4/interfaces/IBundle';
+import { useSelector, useDispatch } from 'react-redux';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import {
+  locationHierarchyResourceType,
+  convertApiResToTree,
+  Tree,
+  TreeNode,
+  locationTreeStateDucks,
+} from '@opensrp/fhir-location-management';
+
+const { reducerName, reducer, setSelectedNode, getSelectedNode } = locationTreeStateDucks;
+
+reducerRegistry.register(reducerName, reducer);
+
+interface LocationUnitListProps {
+  fhirBaseURL: string;
+  fhirRootLocationIdentifier: string;
+}
+
+export const AffiliationList: React.FC<LocationUnitListProps> = (props: LocationUnitListProps) => {
+  const { fhirBaseURL, fhirRootLocationIdentifier } = props;
+  const selectedNode = useSelector((state) => getSelectedNode(state));
+  const dispatch = useDispatch();
+
+  const hierarchyParams = {
+    identifier: fhirRootLocationIdentifier,
+  };
+
+  // get the root locations. the root node is the opensrp root location, its immediate children
+  // are the user-defined root locations.
+  const {
+    data: treeData,
+    isLoading: treeIsLoading,
+    error: treeError,
+  } = useQuery<IBundle | undefined, Error, TreeNode | undefined>(
+    [locationHierarchyResourceType, hierarchyParams],
+    async () => {
+      return new FHIRServiceClass<IBundle>(fhirBaseURL, locationHierarchyResourceType).list(
+        hierarchyParams
+      );
+    },
+    {
+      select: (res) => {
+        return res && convertApiResToTree(res);
```

Comment:
doesn't this return a boolean?

Reply:
not necessarily only if res is already a boolean specifically false., if res is null or undefined then the response is also null or undefined respectively. I do not think the res will ever be false, though even if it does happen, we the page will show a `Resource404`